CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Changelog for NeoFS Node
 - Open FSTree in sync mode by default (#1992)
 - `neofs-cli container nodes`'s output (#1991)
 - Do not panic with bad inputs for `GET_RANGE` (#2007)
+- Big objects removal (#1699)
```

Comment:
Could you be a bit more specific here? We have many similar bugs regarding big object removal.

Reply:
sure, expanded